src/Components/Web.JS/src/Rendering/JSRootComponents.ts

```diff
@@ -137,6 +149,9 @@ export function enableJSRootComponents(
       initializerFunc(componentIdentifier, parameters);
     }
   }
+  if(initializationCallbacks.initialized){
+    initializationCallbacks.initialized();
+  }
```

Comment:
If accepting the suggestion above, this reduces to:

```suggestion
initializerResolver();
```